### PR TITLE
Onboard MTC for Hypershift testing

### PR DESCRIPTION
**Pull request by liswang89**

### CODE REVIEW

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml
Comment (Simranpal) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml:57]: based on last rehearsal - 'Cluster version 4.13 not found for stream nightly'
should the config be updated?

**Review by Simranpal — COMMENTED** [review]
Will there be onboarding for 4.15 as well?
Also 1 comment below

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml
Comment (calebevans) [reply ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml:57]: @Simranpal we have some discussion regarding this issue in [this thread](https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1708975693481159), if you're interested

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

File: ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml
Comment (calebevans) [inline ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa-hypershift.yaml:78]:
```suggestion
      TEST_PLATFORM: hypershift
```

Regarding the error you sent me in slack (`find: ‘tmp/clusters-data/rosa’: No such file or directory`). This is because the following lines are using the `TEST_PLATFORM` variable, and the openshift-cli-installer has set that to `hypershift`.

[Here is the code](https://github.com/openshift/release/blob/master/ci-operator/step-registry/mtc/prepare-clusters/mtc-prepare-clusters-commands.sh#L7-L9) that the error comes from:
```
tar -xzvf "${SHARED_DIR}/clusters_data.tar.gz" --one-top-leve=/tmp/clusters-data

SOURCE_CLUSTER_DIR=$(find tmp/clusters-data/${TEST_PLATFORM} -type d -name "${SOURCE_CLUSTER_PREFIX}*")
```

In the logs, the output shows the tar extraction, then it fails setting the `SOURCE_CLUSTER_DIR` variable. I know that `TEST_PLATFORM` should be `hypershift` because of the tar extraction logs:

```
Extracting cluster data
./
./hypershift/
```

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by deepsm007** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by Sau1506mya** [comment]
@liswang89 , Can you please update the config with firewatch `success_rules` and also add `reporter_config` in periodics file for the slack integration.

**Comment by Sau1506mya** [comment]
Also it will be good if you trigger the job for interop and rosa-classic as well since registry content changed. So, checking here will be good before merging. Thoughts?

**Comment by liswang89** [comment]
/pj-rehearse  periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.16-lp-interop-mtc-interop-aws

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-hypershift-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
@Sau1506mya Rehearsals on hypershift and interop passed. For Rosa Classic, there is a OCM log issue that is blocked by rosa team. Do we prefer to merge the PR and get back if there is rosa issue?

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Comment by stillalearner** [comment]
/pj-rehearse periodic-ci-mtc-qe-mtc-e2e-qev2-master-mtc1.8-ocp4.14-lp-rosa-classic-mtc-interop-aws-rosa

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**
